crates/rust-project-goals/src/gh/labels.rs

```diff
@@ -13,18 +13,30 @@ pub struct GhLabel {
 
 impl GhLabel {
     pub fn list(repository: &Repository) -> Result<Vec<GhLabel>> {
-        let output = Command::new("gh")
-            .arg("-R")
-            .arg(&repository.to_string())
-            .arg("label")
-            .arg("list")
-            .arg("--json")
-            .arg("name,color")
-            .output()?;
+        let mut limit = 128;
+
+        loop {
+            let output = Command::new("gh")
+                .arg("-R")
+                .arg(&repository.to_string())
+                .arg("label")
+                .arg("list")
+                .arg("--json")
+                .arg("name,color")
+                .arg("-L")
+                .arg(format!("{limit}"))
+                .output()?;
```

Comment:
This also needs to check for the command's success as well, this will only check that the `gh` process successfully spawned, but not that the command, and GH's server reply, was successful. There's an example just below, for the `create` function.

This issue was sprinkled throughout the codebase, but I didn't have the time to fix it everywhere in https://github.com/rust-lang/rust-project-goals/pull/384/commits/081512dc70c6e5237180f6f55b2bf005c3fb20c1